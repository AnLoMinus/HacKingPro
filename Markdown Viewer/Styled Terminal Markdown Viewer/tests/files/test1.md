# header 1
## header 2
### header 3
#### header 4

- List 1
- List 2

fenced code

```
class C1(object):
    pass
```

indent code

    class C1(object):
        pass





