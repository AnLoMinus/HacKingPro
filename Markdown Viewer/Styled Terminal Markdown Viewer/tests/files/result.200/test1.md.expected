
[38;5;59m[0m[38;5;209mheader 1[0m

[38;5;59m[0m[38;5;74m header 2[0m

[38;5;59m[0m[38;5;74m  header 3[0m

[38;5;59m[0m[38;5;167m   header 4[0m
[38;5;188m    [38;5;209m- [0mList 1[0m
[38;5;188m    [38;5;209m- [0mList 2[0m
[38;5;188m  fenced code[0m
[38;5;188m  
  [38;5;59m| [0m [38;5;102m[38;5;74mclass[0m[38;5;102m [0m[38;5;102mC1[0m[38;5;102m([0m[38;5;102mobject[0m[38;5;102m)[0m[38;5;102m:[0m[38;5;102m
  [38;5;59m| [0m [38;5;102m[0m[38;5;102m    [0m[38;5;74mpass[0m[38;5;102m
[0m[0m
[38;5;188m  indent code[0m

  [38;5;59m| [0m [38;5;102m[38;5;74mclass[0m[38;5;102m [0m[38;5;102mC1[0m[38;5;102m([0m[38;5;102mobject[0m[38;5;102m)[0m[38;5;102m:[0m[38;5;102m
  [38;5;59m| [0m [38;5;102m[0m[38;5;102m    [0m[38;5;74mpass[0m[38;5;102m
[0m


